habitat-lab/habitat/utils/geometry_utils.py

```diff
@@ -115,3 +116,53 @@ def agent_state_target2ref(
     )
 
     return (rotation_in_ref_coordinate, position_in_ref_coordinate)
+
+
+def random_triangle_point(
+    v0: np.ndarray, v1: np.ndarray, v2: np.ndarray
+) -> np.ndarray:
+    """
+    Sample a random point from a triangle given its vertices.
+    """
+
+    # reference: https://mathworld.wolfram.com/TrianglePointPicking.html
+    coef1 = random.random()
+    coef2 = random.random()
+    if coef1 + coef2 >= 1:
+        # transform "outside" points back inside
+        coef1 = 1 - coef1
+        coef2 = 1 - coef2
+    return v0 + coef1 * (v1 - v0) + coef2 * (v2 - v0)
+
+
+def point_in_triangle_test(
```

Comment:
is test the best name for this method. It sounds too much like a unit_test to me...

Reply:
is_point_in_triangle ?